src/js/components/Image/Image.js

```diff
@@ -4,16 +4,24 @@ import { ImagePropTypes } from './propTypes';
 
 const Image = forwardRef(
   ({ a11yTitle, fallback, onError, opacity, fill, src, ...rest }, ref) => {
-    const [imageMissing, setImageMissing] = useState(false);
-    const handleError = (event) => {
-      if (onError) {
-        onError(event);
-      }
-      setImageMissing(true);
+    const [isFallbackInUse, setFallbackFlag] = useState(false);
+    
+    /* eslint-disable no-param-reassign */
+    const handleError = (event)=>{
+      if(!isFallbackInUse)
+        event.target.src=fallback;
+      setFallbackFlag(true);
```

Comment:
```suggestion
    const handleError = (event)=>{
      if (onError)
        onError(event);
      if(!isFallbackInUse && fallback && fallback !== ""){
        // eslint-disable-next-line no-param-reassign
        event.target.src=fallback;
        setFallbackFlag(true);
      }
```
I think we should still include `onError`. We should also check to make sure fallback is defined and it is not an empty string.